src/deepsparse/tasks.py

```diff
@@ -120,13 +120,18 @@ class SupportedTasks:
         ),
     )
 
+    vit_pose = namedtuple("vit_pose", ["vit_pose"])(
```

Comment:
need to remove these changes since we moved to the examples directory.

for use, users will need to import examples/vit_pose before calling `Pipeline.create`